File: ruby/6_kyu/cw_findoddint.rb
def find_it(seq)
  count = Hash.new(0)
  seq.each { |num| count[num] += 1 }
  count.each { |k, v| return k if v % 2 == 1 }
end

#https://www.codewars.com/kata/54da5a58ea159efa3800083690

File: javascript/6_kyu/cw_findoddint.js
function findOdd(A) {
  let count = {}
  for (i = 0; i < A.length; i++) {
    (count[A[i]] === undefined) ? count[A[i]] = 1 : count[A[i]] += 1;
  }
  let idx = 0;
  let val = Object.values(count);
  for (j = 0; j < val.length; j++) {
    if (val[j] % 2 === 1) {
      idx = j
    }
  }
  return Number(Object.keys(count)[idx]);
}

// https://www.codewars.com/kata/54da5a58ea159efa38000836